src/components/pages/AdminTools/ManageUsers.js

```diff
@@ -271,13 +276,13 @@ const ManageUsersPage = props => {
               </Form.Item>
               <Radio.Group
                 onChange={onChange}
-                name="role"
-                value={formValues.role}
+                name="role_id"
+                value={formValues.role_id}
                 className="radio"
               >
-                <Radio value="user">User</Radio>
-                <Radio value="moderator">Moderator</Radio>
-                <Radio value="admin">Admin</Radio>
+                <Radio value="3">User</Radio>
+                <Radio value="2">Moderator</Radio>
+                <Radio value="1">Admin</Radio>

```

Comment:
A better solution here might be pulling the role values and names from the backend and rendering them in a forEach or map, so the code wouldn't break if more roles were added or the names were changed.